Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Button.cs

```diff
@@ -179,6 +187,19 @@ protected override void HandleUpdate()
             }
         }
 
+        /*
```

Comment:
Exactly, I would like to leave it in for others to continue. I got pretty far already.